nifi-pravega-processors/src/main/java/org/apache/nifi/processors/pravega/AbstractPravegaProcessor.java

```diff
@@ -127,6 +128,23 @@ public ValidationResult validate(String subject, String input, ValidationContext
             .defaultValue("1")
             .build();
 
+    static final PropertyDescriptor PROP_KEYCLOAK_JSON = new PropertyDescriptor.Builder()
+            .name("keycloakJson")
+            .displayName("Keycloak Json")
+            .description("The Keycloak Json.")
+            .required(false)
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .build();
+
+    static final PropertyDescriptor PROP_LOCAL_PRAVEGA = new PropertyDescriptor.Builder()
+            .name("PROP_LOCAL_PRAVEGA")
+            .displayName("Local Pravega (true/false)")
```

Comment:
Rename this to "Create Scope" and PROP_CREATE_SCOPE. here and throughput this PR. Default should be true.

Reply:
Changed